src/context/CartContext.tsx

```diff
@@ -33,6 +33,21 @@ const deleteProduct = (list: Array<IProductShoppingCart>, id: string) => {
   return { newList: newList, qtyDeleted: deleted.quantity };
 };
 
+const changeQuantity = (
+  id: string,
+  quantity: number,
+  list: Array<IProductShoppingCart>
+) => {
+  let oldQty;
+  list.forEach((element) => {
```

Comment:
I think that the loop should be done using a while/for structure and it must break the loop when the desired element is found.